rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" category-id="optional">
+                 <hint title="Dynamic class instantiation" effort="1" category-id="optional">
```

Comment:
Well, it seems to like it is more than Optional as Potential issue category-id from this and previous discussions. Sometimes there it is a problem, it could have higher effort than 1 anyway.
Btw, everybody can take an existent rule and overwrite (copy it into your own location and keep the id) its properties (effort, titles, etc). We have this feature from 2.6.0.Final - 
look at https://access.redhat.com/documentation/en/red-hat-jboss-migration-toolkit/2.7/paged/windup-rules-development-guide/chapter-5-overriding-rules